rust/src/lib.rs

```diff
@@ -825,6 +828,18 @@ pub trait SeleniumManager {
         Ok(driver_path)
     }
 
+    fn stats(&self) {
+        let props = Props {
+            browser: self.get_browser_name().to_string(),
+            browser_version: self.get_browser_version().to_string(),
+            os: self.get_os().to_string(),
+            arch: self.get_arch().to_string(),
+            lang: self.get_language_binding().to_string(),
+            selenium_version: self.get_selenium_version().to_string(),
+        };
+        send_stats_to_plausible(self.get_http_client(), props, self.get_logger());
```

Comment:
>I was not aware it was a requirement. But I can implement it.

Yeah. We did not finish the conversation on #11211 before this PR was created. When implementing tracking, we need to give users the possibility to opt-out. 

>It's a blocking call. Everything happens in the same thread in Selenium Manager. But I can explore how to spawn a new thread for this call.

We need to make it non-blocking because it should not affect the user if this call fails.

Reply:
I have just added two commits to this PR to implement the requested features. Now, the POST request to plausible.io is done asynchronously. Also, the argument `--avoid-stats` is accepted by Selenium Manager for not sending the statistics to plausible.io. As usual, this argument can be configured using the configuration file or an env (`SE_AVOID_STATS=true`).